frontend/server/libs/dao/Users_Badges.dao.php

```diff
@@ -25,4 +25,32 @@ public static function getUserOwnedBadges(Users $user) {
         $args = [$user->user_id];
         return $conn->GetAll($sql, $args);
     }
+
+    public static function getBadgeOwnersCount(string $badge) {
+        global $conn;
+        $sql = 'SELECT
+                    COUNT(*) AS total
+                FROM
+                    Users_Badges
+                WHERE
+                    badge_alias = ?;';
+        $args = [$badge];
+        return $conn->GetRow($sql, $args)['total'];
+    }
+
+    public static function getBadgeFirstAssignationTime(string $badge) {
+        global $conn;
+        $sql = 'SELECT
+                    ub.assignation_time
```

Comment:
```suggestion
                    MIN(ub.assignation_time)
```

así no necesitas el `ORDER BY`.